niftypet/nimpa/acr/proc.py

```diff
@@ -0,0 +1,71 @@
+"""ACR/Jaszczak PET phantom I/O and auxiliary functions"""
+__author__ = "Pawel Markiewicz"
+__copyright__ = "Copyright 2021-23"
+
+import os
+from itertools import chain
+from pathlib import Path, PurePath
+from subprocess import run
+
+import dcm2niix
+from miutil.fdio import hasext
+
+from ..prc import imio, prc
+
+
+def preproc(indat, Cntd, smooth=True, reftrim='', outpath=None, mode='nac'):
+    """Convert to NIfTI (if DICOM), smooth using the Gaussian and trim/scale up."""
+    opth = Path(indat).parent if outpath is None else Path(outpath)
+
+    if mode == 'nac':
+        outdir = opth / mode.upper()
+    elif mode[:3] == 'qnt':
+        outdir = opth / mode.upper()
+    else:
+        raise ValueError('unrecognised mode')
+
+    imio.create_dir(outdir)
+
+    if isinstance(indat, (str, PurePath)) and Path(indat).is_dir():
+        # CONVERT TO NIfTI
+        if not imio.dcmdir(indat):
+            raise IOError('the provided folder does not contain DICOM files')
+        for f in chain(outdir.glob('*.nii*'), outdir.glob('*.json')):
+            # remove previous files
+            os.remove(f)
+        run([dcm2niix.bin, '-i', 'y', '-v', 'n', '-o', outdir, '-f', '%f_%s', str(indat)])
```

Comment:
this was `'f', '%f_%s'` instead of `'-f', '%f_%s'`. Presumably latter is correct.